Content.Shared/_Harmony/Conspirators/Components/ConspiratorComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Content.Shared.StatusIcon;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._Harmony.Conspirators.Components;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ConspiratorComponent : Component
+{
+    [DataField]
+    public ProtoId<FactionIconPrototype> ConspiratorIcon = "ConspiratorFaction";
+}
```

Comment:
Components that are only supposed to be sent to certain people should be set as session specific
```suggestion
public sealed partial class ConspiratorComponent : Component
{
    [DataField]
    public ProtoId<FactionIconPrototype> ConspiratorIcon = "ConspiratorFaction";

    public override bool SessionSpecific => true;
}
```